app/views/mentoring/automation/tooltip_locked.html.haml

```diff
@@ -1,21 +1,20 @@
-.heading.c-generic-tooltip.text-14.w-413
-  %h4.font-semibold.mx-24.mt-20.mb-4 What is automation?
-
-  -# %p.mx-24
-    = ReactComponent::
-    Requires a satisfaction percentage of
-    %strong= @min_satisfaction_percentage.to_s
-    (current: #{@satisfaction_percentage})
-
-  -# %p.mx-24
+.p.heading.c-generic-tooltip.text-14.some{style:'padding:16px 24px;width:413px;'}
+  %p.text-16.text-white.font-semibold.leading-170.mb-12
     Requires
-    %strong= @min_finished_mentoring_sessions.to_s
-    finished mentoring sessions
-    (current: #{@finished_mentoring_sessions})
+    %strong.font-bold= @min_finished_mentoring_sessions.to_s
+    finished mentoring sessions and a satisfaction rating of #{@min_satisfaction_percentage.to_s}% 
+    %br (current rating is #{@satisfaction_percentage}%)
+    
+  .bg-darkThemeBackgroundColor.flex.flex-row.rounded-8.mb-6.overflow-hidden.some{style: 'height:16px;'}
+    .bg-gradient-to-r.from-gradientProgressBarA.to-gradientProgressBarB.rounded-8.some{style:'min-width:6px; height:100%; width:' +(@finished_mentoring_sessions.to_s)+'%'}
```

Comment:
You have `h-100`, `min-w-[6px]` you can use as classes. Then the last thing width should correctly be a style as it includes a variable.